outputs.tf

```
@@ -3,3 +3,8 @@ output "drata_service_account_key" {
   description = "Service Account Key"
   sensitive   = true
 }
+
+output "drata_service_account" {
```

Also update the output name to match the content

```suggestion
output "drata_service_account_email" {
```